### PR TITLE
refactor: update the readme of develop

### PR DESCRIPTION
**Pull request by assynour**
- added link to medium article
- added link to the solution branch
- removed the code license information (since the code is the `solution` branch, it is mentioned in the README of `solution`)
- put the 'How to Run' section before the tutorial content
- minor fixes to the text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,14 +66,4 @@ The following information is visualized:
...
-
-**NOTE**: this tutorial was tested with Node 16 and npm 8.16.0
-
-Run
-```bash
-npm install
-npm start
-```
 
-Open the URL mentioned in the console
```
Comment (tbouffard) [inline README.md:63]: We must keep this, even if it is in the article
This is a common practice to find such commands in the README to easily start the application.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
 
-A getting started demo that uses the [bpmn-visualization](https://github.com/process-analytics/bpmn-visualization-js) library to visualize runtime monitoring data ⏲️.
+This is a template bootstrap. You can clone it and use it to follow the [getting started tutorial](https://medium.com/@process-analytics/getting-started-with-bpmn-visualization-a54fbafae2df).
 
-The code of this tutorial is released under the [Apache 2.0](LICENSE) license.
+The tutorial uses the [bpmn-visualization](https://github.com/process-analytics/bpmn-visualization-js) library to visualize runtime monitoring data ⏲️.
 
-## Content
-The following information is visualized:
+The solution is available in the [solution](https://github.com/process-analytics/bpmn-visualization-tutorial-getting-started/tree/solution) branch.
```
Comment (tbouffard) [inline README.md]:
```suggestion
The whole solution is available in the [solution](https://github.com/process-analytics/bpmn-visualization-tutorial-getting-started/tree/solution) branch.

```

**Review by tbouffard — COMMENTED** [review]
LGTM, the medium is not already published but that's fine.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,27 @@
 # bpmn-visualization 'getting started' tutorial
 
-A getting started demo that uses the [bpmn-visualization](https://github.com/process-analytics/bpmn-visualization-js) library to visualize runtime monitoring data ⏲️.
+This is a template bootstrap. You can clone it and use it to follow the [getting started tutorial](https://medium.com/@process-analytics/getting-started-with-bpmn-visualization-a54fbafae2df).
 
-The code of this tutorial is released under the [Apache 2.0](LICENSE) license.
```
Comment (tbouffard) [inline README.md:5]: We must keep this information. The README is displayed in the front page of the repository on GitHub and this is what the user see when she clones/retrieves the repository locally.

Comment (assynour) [reply README.md:5]: fixed in 20ed06b8c394193819decc5d6e2c51de18add023

**Review by tbouffard — APPROVED** [review]
LGTM, thanks

**Pull request merged**